### DISCUSSION TITLE
Geocode for Overture Maps Foundation

### DISCUSSION THREAD

**Discussion by roboes** [geopy/geopy#554] [Ideas]
Hi,

Yesterday the the Overture Maps Foundation - which includes Amazon, Meta, Microsoft, and TomTom - released its first open map dataset (Source: [Overture Maps Foundation Releases Its First World-Wide Open Map Dataset](https://overturemaps.org/overture-maps-foundation-releases-first-world-wide-open-map-dataset/)).

Overture Maps data is available in cloud-native [Parquet](https://parquet.apache.org/docs/) format (Source: [Overture Maps Data Repo](https://github.com/OvertureMaps/data#accessing-overture-maps-data)).

It would be great to have it implemented as a [Geocoder](https://geopy.readthedocs.io/en/stable/#module-geopy.geocoders) in geopy.

**Comment by pol-todo** [comment]
+1 to this!

**Comment by naimkhrof** [comment]
+1

**Comment by JimCallahanOrlando** [comment]
EXAMPLE USING PYTHON WHEREABOUTS?
Perhaps a Geocoding example using the Python geocoding library WHEREABOUTS
WHEREABOUTS -- Whereabouts is an open-source (MIT licensed) geocoding library for Python which runs locally using DuckDB.
https://github.com/ajl2718/whereabouts

Rather than using a pre-built database, you can create your own geocoder database if you have your own address file. 
https://github.com/ajl2718/whereabouts?tab=readme-ov-file#option-2-create-a-geocoder-database

```
PARQUET FILE
This file should be a single csv or parquet file with the following columns:

Column name	Description	Data type
ADDRESS_DETAIL_PID	Unique identifier for address	int
ADDRESS_LABEL	The full address	str
ADDRESS_SITE_NAME	Name of the site. This is usually null	str
LOCALITY_NAME	Name of the suburb or locality	str
POSTCODE	Postcode of address	int
STATE	The state, region or territory for the address	str
LATITUDE	Latitude of geocoded address	float
LONGITUDE	Longitude of geocoded address	float

```
So, the example would:
1. Download Overture Maps foundation Addresses  for a region,
2. Create a  parquet file usable by Whereabouts,
4. Demonstrate using Whereabouts In Python  (with the new parquet file) for geocoding a user address file.
